### PR TITLE
[author] Update Day.js npm auto-update, repo url w/ the latest v1.6.9

### PR DESCRIPTION
**Pull request by iamkun**
Pull request for issue: #12828
Related issue(s): #12828

From dayjs@1.6.0, we changed the structure of our project.

We moved dayjs.min.js from `root/dist/dayjs.min.js` to` root/dayjs.min.js`.

And according to @PeterDaveHello 's note, I only updated the repo url and auto-update.

Here is the pull request to update this changing.

Also update the repo url since `https://github.com/xx45/dayjs.git` will redirect to `https://github.com/iamkun/dayjs.git`

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Essential checklist
 * [x] I'm the author of this library

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by iamkun** [comment]
any update on this PR please? @sashberd @PeterDaveHello

File: ajax/libs/dayjs/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "dayjs",
   "filename": "dayjs.min.js",
-  "version": "1.5.24",
```
Comment (extend1994) [inline ajax/libs/dayjs/package.json]: According to https://github.com/cdnjs/cdnjs/pull/12835#pullrequestreview-129262487,
You can leave `version` unmodified and don't need to submit `1.6.9/dayjs.min.js` as a result.
(`version` and `filename` changes are followed by submitted files.)

File: ajax/libs/dayjs/package.json [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
+    "url": "https://github.com/iamkun/dayjs.git"
   },
   "npmName": "dayjs",
   "npmFileMap": [
     {
-      "basePath": "dist",
+      "basePath": "",
       "files": [
-        "**/*"
+        "dayjs.*"
```
Comment (extend1994) [inline ajax/libs/dayjs/package.json]: Suppose users need the files under `locale` and `plugins` to extend the functionality of day.js.
You can use [the pattern](https://github.com/cdnjs/cdnjs/blob/b7fbff2ad7f9021f5a9f09357a5061a61d031e13/ajax/libs/dayjs/package.json#L24-L26) you once used :)

If you did the package.json changes locally (with Git commands),
let's separate auto-update config update and repo url update, thanks!

**Comment by iamkun** [comment]
Don't know how to do. I'll close this PR and wait for some help if you have time. Thanks.

**Comment by extend1994** [comment]
I can send a PR for it later :)

**Comment by extend1994** [comment]
@iamkun May I know do users need the files under `locale` and `plugins`? Not sure my guessing is correct.

**Comment by iamkun** [comment]
@extend1994 Thanks, Yes, users need these files.
`root/dayjs.min.js` is the main bundle file. 
`root/plugin/*` `root/locale/*` are plugins and i18n files.

**Comment by PeterDaveHello** [comment]
@iamkun we're happy to help, sorry for the delay

**Comment by iamkun** [comment]
@PeterDaveHello Thanks for your help. Cheers.

**Comment by extend1994** [comment]
PR is opened #12882

**Pull request closed without merging**